_organizers/aditya-dhoot.md

```diff
@@ -0,0 +1,10 @@
+---
+github: adityadhoot
+layout: base
+name: Aditya Dhoot
+photo_url: /static/img/organizers/adityad.jpg
```

Comment:
I'm reviewing from my phone so I can't run this, but it's safer to make sure the casing matches on the file.

Reply:
fixed